src/Groups/matrices/iso_nf_fq.jl

```diff
@@ -0,0 +1,154 @@
+# Detinko, Flannery, O'Brien "Recognizing finite matrix groups over infinite
+# fields", Section 4.2
+function isomorphic_group_over_finite_field(matrices::Vector{T}) where T <: Union{ fmpq_mat, AbstractAlgebra.MatElem{nf_elem} }
```

Comment:
```suggestion
function isomorphic_group_over_finite_field(matrices::Vector{T}) where T <: Union{fmpq_mat, AbstractAlgebra.MatElem{nf_elem}}
```
Also, there is a common overtype for matrices with entries in `fmpq`. Can you use this? I think it is `MatrixElem{fmpq}`. It should be a union of two things.

Reply:
Oh, wait, did you also want me to remove the spaces? Is there a "style guide" for Oscar somewhere?

Reply:
Yes and no :)